### ISSUE TITLE
Console patching interfering with CLI

### ISSUE THREAD

**Issue by sdgluck** [vadimdemedes/ink#114]
Related to #83.

The patching of console is causing an issue for me. My CLI patches console already for its own purposes, and Ink's patching interferes with this.

At the moment I am reversing the patches on console methods after calling `render` for the first time, which you can see here: https://github.com/wpk-cli/wpk/blob/ink/packages/command-build/src/terminal/index.js#L55

Is there some way Ink could accept an option that when set prevents it from touching `console`? Happy to put in a PR.

Thanks for a fantastic library. 👍

**Comment by vadimdemedes** [comment]
Glad you're enjoying Ink! I think you'll enjoy Ink 2 even more, since it doesn't have any kind of console patching :) You can check the docs out here https://github.com/vadimdemedes/ink/tree/next and install it via `npm install ink@next`.

**Comment by sdgluck** [comment]
Thanks @vadimdemedes. I updated to `ink@next` however `ink-redux` is not working now:

```sh
  TypeError: Class extends value undefined is not a constructor or null
      at Object.<anonymous> (...\wpk-cli\wpk\packages\command-build\node_modules\ink-redux\dist\index.js:6:24)
      ...
```

Looks like some small changes are needed to use `React.Component` instead of `ink.Component`. I did this myself locally but got an error where the connect HOC tries to get the store off the context. Not sure what's going on there.

For now I'll stick with patching on `ink@latest`.
